src/tv2-common/parts/effekt.ts

```diff
@@ -51,15 +52,34 @@ export function CreateEffektForPartBase(
 		)
 
 		return ret ?? {}
-	} else if (transition !== undefined && transition.duration !== undefined) {
-		if (transition.style.match(/mix/i)) {
-			return CreateMixForPartInner(pieces, partDefinition.externalId, transition.duration, layers) ?? {}
-		} else {
-			return {}
-		}
-	} else {
+	}
+
+	if (transition === undefined || transition.duration === undefined) {
 		return {}
 	}
+
+	if (transition.style.match(/mix/i)) {
+		return (
```

Comment:
Parantheses are unnecessary :) Same for the return for dip.

Reply:
Parantheses are added by tslint... :(